eras/shelley/impl/src/Cardano/Ledger/Shelley/Scripts.hs

```diff
@@ -13,15 +13,17 @@
 {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE UndecidableSuperClasses #-}
+{-# LANGUAGE ViewPatterns #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 module Cardano.Ledger.Shelley.Scripts (
-  MultiSig (
-    RequireAllOf,
-    RequireAnyOf,
-    RequireSignature,
-    RequireMOf
-  ),
+  MultiSig (..),
```

Comment:
You can't export raw constructors, because that will expose the `MultiSigConstr`
```suggestion
  MultiSig,
```